<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigClass</class>
 <widget class="QWidget" name="ConfigClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Config</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>90</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>sourceFolder:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>90</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>destFolder:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_sourceFolder">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>20</y>
     <width>240</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_destFolder">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>50</y>
     <width>240</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_browseSourceFolder">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>20</y>
     <width>75</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_browseDestFolder">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>50</y>
     <width>75</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse...</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
